app/controllers/collection_items_controller.rb

```diff
@@ -1,9 +1,8 @@
 class CollectionItemsController < ApplicationController
   before_action :load_collection
   before_action :load_user, only: [:update_multiple]
-  before_action :load_item_and_collection, only: [:destroy]
+  before_action :load_item_and_collection, only: [:update_multiple]
```

Comment:
Your first instinct was right. This should be removed, and so should `load_item_and_collection`.